ext/ffi/ffi.c

```diff
@@ -119,6 +119,7 @@ struct _zend_ffi_type {
 			ffi_abi        abi;
 		} func;
 	};
+	bool sized;
```

Comment:
Please don't introduce new field. Use ZEND_FFI_ATTR_INCOMPLETE_ARRAY and ZEND_FFI_ATTR_VLA attribute. Or add a new attribute ZEND_FFI_ATTR_UNKNOWN_SIZE (may be with a better name).